src/journal/journald.c

```diff
@@ -52,7 +52,7 @@ int main(int argc, char *argv[]) {
                 goto finish;
 
         server_vacuum(&server, false);
-        server_flush_to_var(&server);
+        server_flush_to_var(&server, true);
```

Comment:
> This changes journald to not write to /var/log/journal until it received
SIGUSR1 for the first time, thus having been requested to flush the runtime
journal to disk

Hm, this is misleading. `SIGUSR1` is the one way.
Another way:
```
# Storage=auto, /var/log/journal doesn't exist
echo Storage=persistent >>/etc/systemd/jouranld.conf
touch /run/systemd/journal/flushed # systemd-journal-flush.service runs journalctl --flush, --flush creates  /run/systemd/journal/flushed
systemctl restart systemd-journald
```
So, I don't really need `SIGUSR1`

> Fixes: #1397

Hm, but we can't fix "broken" acl/permissions: https://github.com/systemd/systemd/issues/1397#issuecomment-160497885
Or should I file another issue for that?